src/sentry/notifications/platform/sample.py

```diff
@@ -30,20 +30,17 @@ class ErrorAlertData(NotificationData):
 @template_registry.register(ErrorAlertData.source)
 class ErrorAlertNotificationTemplate(NotificationTemplate[ErrorAlertData]):
     category = NotificationCategory.DEBUG
-
-    @property
-    def example_data(self) -> ErrorAlertData:
-        return ErrorAlertData(
-            error_type="ValueError",
-            error_message="'NoneType' object has no attribute 'get'",
-            project_name="my-app",
-            issue_id="12345",
-            error_count=15,
-            first_seen="2024-01-15 14:30:22 UTC",
-            chart_url="https://example.com/chart",
-            issue_url="https://example.com/issues",
-            assign_url="https://example.com/assign",
-        )
+    example_data = ErrorAlertData(
```

Comment:
the service expects raw access to `data.source`. When it tries to access `example_data.source` the `@property` decorator causes issues. But also, this is just less work for devs adding notifications anyway.